content/webapp/components/EventSchedule/EventScheduleItem.tsx

```diff
@@ -98,30 +98,54 @@ const eventLocations = (locations: Place[], isHybridEvent: boolean) => {
 
 // We have a message block on scheduled events which either displays
 // 'Just turn up' or 'Arrive early to register'
-// We only show this message if:
-// - the event isn't past AND
-// - it doesn't require booking (either through Eventbrite or by contacting the booking enquiry team) AND
-// - it doesn't have it's own event schedule AND
-// - the parent event isn't Ticketed
-function shouldShowMessage({
-  event,
-  parentEvent,
-}: {
-  event: Event;
-  parentEvent: Event;
-}): boolean {
+// N.B. If the criteria to display both messages is satisfied, we only show 'Arrive early to register' (see below)
+// We don't show either message if the following criteria aren't satisfied:
+function shouldShowMessage(event: Event): boolean {
   return (
     !isEventPast(event) &&
     !event.eventbriteId &&
     !event.bookingEnquiryTeam &&
-    !(event.schedule && event.schedule.length > 1) &&
-    parentEvent.bookingType !== 'Ticketed'
+    !(event.schedule && event.schedule.length > 1)
+  );
+}
+// We should show the 'Arrive early to register' message if:
+// - event.hasEarlyRegistration is true AND
+// - the event isn't past AND
+// - it doesn't require booking (either through Eventbrite or by contacting the booking enquiry team) AND
+// - it doesn't have it's own event schedule
+function shouldShowEarlyRegistrationMessage(event: Event): boolean {
+  return event.hasEarlyRegistration;
+}
```

Comment:
Just tested adding "no" to the dropdown, and as this only checks if the field is _there_, it still returns true. So maybe we should check

`return event.hasEarlyRegistration === 'yes'` 

until we can use a boolean toggle

Reply:
I've added this in the Event transformer so hasEarlyRegistration is still a boolean.